src/chart/pie/pieLayout.ts

```diff
@@ -91,7 +93,10 @@ export default function pieLayout(
 
         const { cx, cy, r, r0 } = getBasicPieLayout(seriesModel, api);
 
-        const startAngle = -seriesModel.get('startAngle') * RADIAN;
+        let startAngle = -seriesModel.get('startAngle') * RADIAN;
+        let endAngle = startAngle - PI2;
+
+        const endAngleModel = seriesModel.get('endAngle');
```

Comment:
The result of `seriesModel.get('endAngle')` should not be named as `xxxModel` because it's not a model.

Reply:
I would suggest:

```ts
let endAngle = seriesModel.get('endAngle'`);
endAngle = endAngle === 'auto' ? startAngle - PI2 : -endAngle * RADIAN;

Reply:
fix endAngle.